docs/resources/machine_pool.md

```diff
@@ -23,11 +23,11 @@ Machine pool.
 
 ### Optional
 
-- `autoscaling_enabled` (Boolean) Enables autoscaling.
-- `labels` (Map of String) Labels for machine pool. Format should be a comma-separated list of 'key = value'. This list will overwrite any modifications made to node labels on an ongoing basis..
-- `max_replicas` (Number) Max replicas.
+- `autoscaling_enabled` (Boolean) Enables autoscaling. This variable requires you to set a maximum and minimum replicas range using the `max_replicas` and `min_replicas` variables.
+- `labels` (Map of String) Labels for the machine pool. Format should be a comma-separated list of 'key = value'. This list will overwrite any modifications made to node labels on an ongoing basis.
+- `max_replicas` (Number) The maximum number of replicas not exceeded by the autoscaling functionality.
 - `max_spot_price` (Number) Max Spot price.
```

Comment:
Not sure if "Max" needs to be spelled out as "maximum". Also unsure why "spot" is capitalized.